aicsimageio/tests/test_aics_image.py

```diff
@@ -237,11 +256,137 @@ def test_multi_scene_aicsimage(
     # Run checks
     run_multi_scene_image_read_checks(
         ImageContainer=AICSImage,
-        uri=uri,
+        image=uri,
         first_scene_id=first_scene_id,
         first_scene_shape=first_scene_shape,
         first_scene_dtype=np.uint16,
         second_scene_id=second_scene_id,
         second_scene_shape=second_scene_shape,
         second_scene_dtype=np.uint16,
     )
+
+
+@pytest.mark.parametrize(
+    "array_constructor",
+    [
+        # np.ones,
+        da.ones,
+    ],
+)
```

Comment:
FWIW if this is a really common test parameterization, you can set it to a variable somewhere in the testing module and use that instead:
```python
array_constructors = pytest.mark.parametrize("array_constructor", [np.ones, da.ones])

...

# Test that uses this
@array_constructor
@pytest.mark.parametrize(...)
def test_thing():
```

Reply:
Neato! Thanks.

Reply:
Didn't do it for this PR but probably will later if it _keeps_ getting used.